packages/vue-router/src/types.ts

```diff
@@ -63,7 +63,7 @@ export interface ViewItem {
   outletId: number;
   matchedRoute: RouteLocationMatched;
   ionPageElement?: HTMLElement;
-  vueComponent: any; // todo
+  vueComponent: any;
```

Comment:
Similar question here regarding removing the todo for something with an `any` type.